### PR TITLE
Add Birdy

### PR DESCRIPTION
**Pull request by Zeitsperre**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by cehbrecht** [comment]
@Zeitsperre Please add me as maintainer to the birdy package.

**Comment by cehbrecht** [comment]
@Zeitsperre I have build with this recipe locally and skipped the `pytest` tests ... just kept the `import` test ... and made a local installation. It worked for me. I didn't notice issues with `python-dateutil` ... it is a dependency by owslib which was installed. Do you have a test case?

File: recipes/birdy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+build:
+  skip: true  # [win or py2k]
+  number: 0
+  script: python -m pip install . --no-deps -vv
+  entry_points:
+    - birdy = birdy.cli.run:cli
+
+requirements:
+  host:
+    - python >=3
```
Comment (marcelotrevisani) [inline recipes/birdy/meta.yaml]: As you are skipping the build for ``python 2`` you can remove this restriction here
```suggestion
    - python
```

File: recipes/birdy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - six
+    - click
+    - lxml
+    - owslib >=0.18.0
+    - jinja2
+    - wrapt
+    - funcsigs
+    - boltons
+  run:
+    - python >=3
```
Comment (marcelotrevisani) [inline recipes/birdy/meta.yaml]:
```suggestion
    - python
```

File: recipes/birdy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bird-house/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ece0aa45130b874ec623b763027e1ee5da125fdbf83a30cdb622ed1d9facd1be
+
+build:
+  skip: true  # [win or py2k]
```
Comment (marcelotrevisani) [inline recipes/birdy/meta.yaml]: why are you skipping ``windows``?

File: recipes/birdy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    - six
+    - click
+    - lxml
+    - owslib >=0.18.0
+    - jinja2
+    - wrapt
+    - funcsigs
+    - boltons
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/birdy/meta.yaml]: You will need to add the restriction here as well
```suggestion
    - python >=3
```

File: recipes/birdy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/bird-house/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ece0aa45130b874ec623b763027e1ee5da125fdbf83a30cdb622ed1d9facd1be
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps -vv
```
Comment (marcelotrevisani) [inline recipes/birdy/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

**Pull request merged**